wishlist.html

```diff
@@ -0,0 +1,135 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Wishlist</title>
+    <link rel="stylesheet" href="./style.css" />
+  </head>
+  <body>
+    <header class="main-head ecom-navbar z-index-lg">
+      <nav class="nav-component">
+        <div class="logo-wrapper">
+          <span class="hamburger-icon"
+            ><i class="fa fa-bars f-6 p-h-4"></i
+          ></span>
+          <h1 class="nav-logo heading-4 p-h-2">Brain Store</h1>
+        </div>
+        <div class="nav-searchBar-container ecom-searchbar">
+          <span class="nav-searchIcon"
+            ><i class="fa fa-search f-5 t-c-3"></i
+          ></span>
+          <input
+            type="text"
+            class="p-v-3 nav-searchBar"
+            placeholder="Search Items..."
+          />
+        </div>
+        <div class="link-wrapper">
+          <a class="t-c-2 f-5 p-v-3 p-h-8 m-h-3 sign-in-button">Sign in</a>
+          <div class="badge-wrapper p-h-4 m-h-3">
+            <span class="badge-icon f-8 m-2">
+              <a href="cart.html" class="t-c-1"
+                ><i class="fa fa-shopping-cart"></i
+              ></a>
+            </span>
+            <div class="badge centered-row ecom-badge">3</div>
+          </div>
+          <span class="f-7 p-h-1 pointer badge-caption">Cart</span>
+        </div>
+      </nav>
+    </header>
+    <p class="heading-3 center-text">My Cart <span class="f-8">(3)</span></p>
```

Comment:
Why not place the text and icon inside the same parent div?
that way user can click on either and still be able to access the page, else this just makes it a label which takes up screen space

Reply:
I am taking a mobile first approach , and in mobile there is no label . I will finalize the desktop CSS in the end.